src/OrchardCore.Modules/OrchardCore.ReCaptcha/Forms/Migrations.cs

```diff
@@ -16,7 +16,7 @@ public Migrations(IContentDefinitionManager contentDefinitionManager)
         public int Create()
         {
             _contentDefinitionManager.AlterPartDefinition("ReCaptchaPart", part => part
-                .WithDescription("Provides captcha properties."));
+                .WithDescription("Provides reCaptcha properties."));
```

Comment:
this will create new sites will `reCaptcha` but existing site still have `captcha `. Also needed to have `Update` migration that updates description for existing sites.